R/common_language.R

```diff
@@ -347,15 +347,16 @@ p_overlap <- function(x,
 # Utils -------------------------------------------------------------------
 
 #' @keywords internal
+#' @import stats quantile optim
```

Comment:
You don't need `@import` if you use `package::fun()`.

Reply:
Yeah, I know, but it feels safer 😅